package.json

```diff
@@ -18,6 +18,7 @@
     "@nomiclabs/hardhat-etherscan": "^2.1.3",
     "@nomiclabs/hardhat-waffle": "^2.0.0",
     "@openzeppelin/contracts": "^4.3.3",
+    "@renproject/gateway-sol": "^1.0.10",
```

Comment:
It seems that this package is causing troubles in CI

https://github.com/Ethernautas/renpool/runs/4314560392?check_suite_focus=true#step:3:20

I believe because the openzeppelin versions used by them and us are different.

Reply:
ok, I'll look into this :+1: